README.md

```diff
@@ -1,11 +1,173 @@
 # Jormungandr
 Main Software Repository for Subbots
 
-To Run UWSim: </br>
-  1: Run install_dependencies.sh script in main repository. </br>
-  2: Add source ~/Jormungandr/devel/setup.sh to .bashrc, or source this file every time. </br>
-  3: Run ~/Jormungandr/src/simulator/run_simulator.sh "scene" (Where "scene" is one of the xml scenes in Jorgumandr/src/simulator/scenes ie. Qualification.xml) </br>
+![alt tag](https://travis-ci.org/ubc-subbots/Jormungandr.svg?branch=master)
 
+## Table of Contents
+- [Setup](#setup)
+  - [Rosinstall](#rosinstall)
+- [Conventions](#conventions)
+  - [Github](#github-conventions)
+  - [Coding Conventions](#coding-conventions)
+  - [Coordinate Systems](#coordinate-systems)
+- [Creating a New Node](#creating-a-new-node)
+- [Creating a New Simple Node](#creating-a-new-simple-node)
+- [Launch Files](#launch-files)
+- [Testing](#testing)
+  - [GTest](#gtest)
+  - [Rostest](#rostest)
+- [Running Packages](#running-packages)
+  - [Running UWSim](#to-run-uwsim)
+  - [Running Phidget IMU](#running-phidget)
+  - [Running Camera Driver](#cammera-driver)
 
-![alt tag](https://travis-ci.org/ubc-subbots/Jormungandr.svg?branch=master)
 
+## Setup 
+
+### Rosinstall
+
+Rostinstall is addressed and run in `install_dependencies.sh`, should there be any need to run manual, rosinstall should be run after all steps from installation and setup has been completed. 
```

Comment:
`Rostinstall` -> `RosInstall`

Reply:
Why?

Reply:
Sorry, the capitalization doesn't really matter, to me, but there's an extra `t` (4th character) in there.

Reply:
Fixed this as well